src/main/java/xyz/nucleoid/plasmid/mixin/fake/PacketByteBufMixin.java

```diff
@@ -18,6 +24,6 @@
             )
     )
     private Item modifyItem(ItemStack stack) {
-        return FakeItem.getProxy(stack.getItem());
+        return Fake.getProxy(stack.getItem());
```

Comment:
This is now essentially duplicated with the `modifyItemStack` mixin

Reply:
`modifyItemStack` doesn't have to change the item type. Maybe it should do that actually 🤔

Reply:
Oh- definitely, lol.